src/components/__snapshots__/Calculator.test.js.snap

```diff
@@ -0,0 +1,226 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Calculator renders correctly 1`] = `
+<DocumentFragment>
```

Comment:
- [ ] Great job so far 👏🏻 just one last thing, 🤔  There should be the usage of Jest snapshots for testing `all the components.` this includes (Home, Quotes, and Buton) components. Let's address this. ✍🏻